py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/convert.py

```diff
@@ -0,0 +1,75 @@
+import gzip
+import json
+import pyarrow as pa
+
+from datetime import datetime
+from multiprocessing import Pool
+from pathlib import Path
+from .configuration import Configuration
+
+# TODO this is fine for now, but maybe an environ variable?
+MULTIPROCESSING_POOL_SIZE = 4
+
+def gz_to_pyarrow(filename: Path, config: Configuration):
```

Comment:
i ended up modifying this function a bit. it no longer takes in the return queue object.

i wasn't able to get these jobs to trigger by calling `return_que.get` inside of the for loop below.

without that return queue, this function returns either tables or filenames.